Pester.psd1

```diff
@@ -111,13 +111,13 @@ PrivateData = @{
         Tags = @('powershell','unit_testing','bdd','tdd','mocking')
 
         # The web address of an icon which can be used in galleries to represent this module
-        IconUri = "http://pesterbdd.com/images/Pester.png"
+        IconUri = "https://pesterbdd.com/images/Pester.png"
```

Comment:
@ChrisUbben, please update your pull request - the pesterbdd.com will not be available via https soon.